### PR TITLE
Minor modification

### PR DESCRIPTION
**Pull request by rmarzocchi84**
How to dowload and install LWC

Add example using git clone and checkout instead of zip

Add a few clarifications

### CODE REVIEW

File: source/install/linux.rst [hunk 1]
```diff
@@ -281,20 +278,44 @@ Retrieve the latest available stable version from our `Github release page <http
...
+
+   
+   # Options
+   VERSION=3.4.0
+   # chose location where create your clone (e.g. /var/www or your home)
+   LOCATION=/var/www 
+   
+   # Clone repository
+   cd $LOCATION
+   git clone https://github.com/3liz/lizmap-web-client.git
```
Comment (Gustry) [inline source/install/linux.rst]: This is not possible with Lizmap 3.4 : https://github.com/3liz/lizmap-web-client/blob/master/INSTALL.md#get-the-source  (I don't think people should build it, they should download packages)

File: source/install/linux.rst [hunk 2]
```diff
@@ -281,20 +278,44 @@ Retrieve the latest available stable version from our `Github release page <http
...
+   # chose location where download your zip (e.g. /var/www or your home)
+   LOCATION=/var/www 
    # Archive recovery with wget
+   cd $LOCATION
    wget https://github.com/3liz/lizmap-web-client/archive/$VERSION.zip
    # Unzip archive
    unzip $VERSION.zip
+
+
+Or clone the repository and than use a specific version (it is a better solution for future update)
```
Comment (Gustry) [inline source/install/linux.rst]: It won't be possible, read the other comment.

**Comment by rmarzocchi84** [comment]
Sorry, I have not yet tested 3.4 version.

Anyway I think it is not a great news, because to use GitHub for update is really useful for production environment.

**Comment by mdouchin** [comment]
There is still the possibility to use Git to fetch LWC, but we think this must not be done on production environment.

For development purpose (or at your own risk), you can still use git and use the `make build` command in LWC root folder to build or update the necessary PHP and JS package for you, anytime you pull from the repository.

**Comment by Gustry** [comment]
Sorry for the late delay.
I have rebased your PR and manually merged it. I can close.

**Pull request closed without merging**